src/ui/utils/formatEstimatedProcessingDuration.ts

```diff
@@ -0,0 +1,8 @@
+import prettyMilliseconds from "pretty-ms";
+
+const MS_IN_MINUTE = 60_000;
+
+export function formatEstimatedProcessingDuration(ms: number) {
+  const minutes = Math.ceil(ms / MS_IN_MINUTE) || 1;
+  return prettyMilliseconds(minutes * MS_IN_MINUTE, { verbose: true });
+}
```

Comment:
This may be controversial, but I think we should avoid showing an estimated processing time that has seconds precision. (We aren't that accurate, and I think it would be misleading to format it that way.) I decided to round up to the nearest minute.